addons/medical/functions/fnc_treatment.sqf

```diff
@@ -201,6 +201,12 @@ _treatmentTime = if (isNumber (_config >> "treatmentTime")) then {
 };
 
 // Start treatment
+if (_caller != ACE_player) exitWith {
+    // It's an AI that is doing the treatment
+    sleep _treatmentTime;
+    [[_caller, _target, _selectionName, _className, _items, _usersOfItems]] call DFUNC(treatment_success);
```

Comment:
Treatment can also still fail at any moment due to any reason. That should also be handled here.